docs/extensibility-http.md

```diff
@@ -293,16 +293,17 @@ Deploy the revised (!) Helm Chart to your cluster:
 HOST="ghcr.io"
 USER="[[GITHUB-USER]]"
 REPO="${HOST}/${USER}"
+VERS="$(cat version.txt)"
 
 sudo microk8s.helm3 install akri ./akri/deployment/helm \
 --set imagePullSecrets[0].name="${HOST}" \
 --set agent.image.repository="${REPO}/agent" \
---set agent.image.tag="v0.0.38-amd64" \
+--set agent.image.tag="${VERS}" \
```

Comment:
one thing to note is that version.txt will have `0.0.38`, not `v0.0.38-amd64`

Reply:
Hmmm .... Should I just revise to say `${VERS}-amd64` and recommend the user replaces with the correct arch?

Reply:
maybe even something like `v${VERS}-amd64` (note the `v`)

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/ecc358b9e2182782abc7e4cedb65041fbb81113a